src/test/aa

```diff
@@ -0,0 +1,112 @@
+//       <input required value="first todo" type="text" class="input-task" placeholder="Add to your list...">
+//       <button type="submit" id="inputButton" class="pointer"><i class="fa fa-level-down fa-rotate-90 enterIcon pointer"></i></button>
+//         </form>
+//         <div class="tasks-cont">
+//         </div>
+//         `;
+  
+//         inputSubmit();
+//       expect(JSON.parse(localStorage.getItem('Task List'))).toEqual(todos);
+//     });
+  
+//     test('add a todo task into the DOM', () => {
+//       document.body.innerHTML = `
+//       <form class="input-cont" onsubmit="inputSubmit(this, event)">
+//       <input required value="first todo" type="text" class="input-task" placeholder="Add to your list...">
+//       <button type="submit" id="inputButton" class="pointer"><i class="fa fa-level-down fa-rotate-90 enterIcon pointer"></i></button>
+//         </form>
+//         <div class="tasks-cont">
+//         </div>
+//         `;
+//         inputSubmit();
+  
+//       expect(document.querySelector('.tasks-cont').children).toHaveLength(2);
+//     });
+//   });
+
+  describe('add items to todoList', () => {
+    test('add a task task to the localStorage', () => {
+      // fill in the localStorage
+      localStorage.setItem('Task List', JSON.stringify([]));
+      // the expected final state of the localStorage
+      const todos = [
+        {
+          description: 'first todo',
+          completed: false,
+          index: 1,
+        },
+        {
+            description: '2nd todo',
+            completed: false,
+            index: 2,
+          },
+          {
+            description: '3rd todo',
+            completed: false,
+            index: 3,
+          },
+          {
+            description: '4th todo',
+            completed: false,
+            index: 4,
+          },
+          {
+            description: '5th todo',
+            completed: false,
+            index: 5,
+          },
+        
+      ];
+  
+      // mock the DOM content
+      todos.map((item) =>{
+      document.body.innerHTML = `
+      <form class="input-cont" onsubmit="inputSubmit(this, event)">
+      <input required value="${item.description}" type="text" class="input-task" placeholder="Add to your list...">
+      <button type="submit" id="inputButton" class="pointer"><i class="fa fa-level-down fa-rotate-90 enterIcon pointer"></i></button>
+        </form>
+        <div class="tasks-cont">
+        </div>
+        `;
+        inputSubmit();
+    });
+      expect(JSON.parse(localStorage.getItem('Task List'))).toEqual(todos);
+      expect(document.querySelector('.tasks-cont').children).toHaveLength(5);
+      splice(1);
+      let taskList= JSON.parse(localStorage.getItem('Task List'));
+      expect(taskList.length).toEqual(4);
+      expect(taskList[0].index).toEqual(1);
+      expect(document.querySelector('.tasks-cont').children).toHaveLength(4);
+    });
+  
+  });
+
```

Comment:
I think you are not using this file so It would be great if you remove it. Please remember that if you are not going to use something It is a good idea to remove it. :nerd_face: :nerd_face: